sdk/gossip/client/roundsubscriber.go

```diff
@@ -168,7 +168,14 @@ func (rs *roundSubscriber) subscribe(ctx context.Context, abr *services.AddBlock
 				RoundConfirmation: noti.RoundConfirmation.Value(),
 			}
 			isDone = true
-			ch <- proof
```

Comment:
eventstream hangs on these functions, so this attempts to send it in the main thread, but if it would hang then it reverts to using a goroutine. I figured in this iteration it would be better to use a goroutine than to let it just drop. We can think about just dropping in the future.